backend/app/database.py

```
@@ -1,7 +1,9 @@
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)